src/client/components/pages/error.js

```diff
@@ -41,12 +41,12 @@ class ErrorPage extends React.Component {
 					<h1>{error.status}</h1>
 				</Row>
 				<Row className="margin-top-6 margin-bottom-2">
-					<b style={{fontSize: '1.2em'}}>
+					<b className="lead">
```

Comment:
Hmm, either just `<p className="lead">` or `<p className="lead"><b>` here - the lead class overrides the font weight styling on the <b> element as it is.